connection/TypeDBConnectionFactory.java

```diff
@@ -1,40 +0,0 @@
-/*
```

Comment:
This class as well as their concrete implementations (for `CoreClient` and `ClusterServerClient`) can now be removed.